### ISSUE TITLE
BatchNorm in FC-DenseNet implementation

### ISSUE THREAD

**Issue by khiemkhanh98** [GeorgeSeif/Semantic-Segmentation-Suite#138]
Hi GerorgeSeif,

It seems that you have implemented Batchnorm in the pre_conv block of the FCN model. However, in your predict.py and model_builder.py files, I don't see you changing the is_training variable of slim.batch_norm to False to enter the Inference mode for FCN model (you did change it for other models like PSP or Deeplab). Can you explain this as I saw on other sources that they said we have to toogle the variable to make the batchnorm layer behave properly?

Also, when I try to change the is_training to False, my testing result is so worst when the whole output map is completely wrong!!. I cannot understand why. The result becomes ok when is_training is set to True during inference.

Can you help me figure this out?

Many thanks

**Comment by GeorgeSeif** [comment]
Hi there @khiemkhanh98 

Hmm you're right, the missing `is_training` for FC DenseNet is a mistake on my part! I'll update that soon. Thanks :)

For which network did you get the poor results when setting `is_training  = False`?

**Comment by khiemkhanh98** [comment]
Hi GeorgeSeif

I'm using my custom model that utilizes some blocks in the FC DenseNet code.
